parallel-consumer-python/pyallel_consumer/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.confluent.parallelconsumer</groupId>
+    <artifactId>python</artifactId>
+    <version>0.1.0a0</version>
+    <properties>
+        <jackson.version>2.13.2</jackson.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.confluent.parallelconsumer</groupId>
+            <artifactId>parallel-consumer-core</artifactId>
+            <version>0.5.2.3</version>
+        </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-avro-serializer</artifactId>
+            <version>5.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-json-serializer</artifactId>
+            <version>7.2.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.13.2.2</version>
```

Comment:
are these and the below, used directly?

Reply:
No, they're used by the Kafka client, but even then probably only because I've setup the examples to use the Kafka JSON serdes
Have included them in the alpha so it's possible to use this with JSON
But would probably need to take similar dependencies for avro/protobuf etc

I don't know how to best handle these optional dependencies
I can setup the python package to have optional dependencies, and if those settings are chosen on install I could cause these to be installed
But that then requires maintaining these optional java dependency mappings in python too
When ideally installing the kafka client would just make it usable with any format

Reply:
If I remove them I get these errors:

Without jackson-core:
```
java.lang.java.lang.ClassNotFoundException: java.lang.ClassNotFoundException: com.fasterxml.jackson.module.paramnames.ParameterNamesModule
```
Without jackson-mapper-asl
```
java.lang.java.lang.NoClassDefFoundError: java.lang.NoClassDefFoundError: com/fasterxml/jackson/core/Versioned
```
Seems I maybe can remove jackson-databind